src/data_provider/src/packages/packagesWindowsParserHelper.h

```diff
@@ -121,6 +123,99 @@ namespace PackageWindowsHelper
         {
         }
     }
+
+    static void getHotFixFromRegWOW(const HKEY key, const std::string& subKey, nlohmann::json& data)
+    {
+        try
+        {
+            std::set<std::string> hotfixes;
+            const auto callback
+            {
+                [&key, &subKey, &hotfixes](const std::string & packageKey)
+                {
+                    const auto callbackKey
+                    {
+                        [&key, &subKey, &packageKey, &hotfixes](const std::string & package)
+                        {
+                            auto hfValue { extractHFValue(package) };
+
+                            if (!hfValue.empty())
+                            {
+                                hotfixes.insert(std::move(hfValue));
+                            }
+                        }
+                    };
+                    Utils::Registry packageReg{key, subKey + "\\" + packageKey, KEY_WOW64_64KEY | KEY_ENUMERATE_SUB_KEYS | KEY_READ};
+                    packageReg.enumerate(callbackKey);
+                }
+            };
+            Utils::Registry root{key, subKey, KEY_WOW64_64KEY | KEY_ENUMERATE_SUB_KEYS | KEY_READ};
+            root.enumerate(callback);
+
+            for (auto& hotfix : hotfixes)
+            {
+                nlohmann::json hotfixValue;
+                hotfixValue["hotfix"] = std::move(hotfix);
+                data.push_back(std::move(hotfixValue));
+            }
+        }
+        catch (...)
+        {
+        }
+
+    }
+
+    static void getHotFixFromRegProduct(const HKEY key, const std::string& subKey, nlohmann::json& data)
+    {
+        try
+        {
+            std::set<std::string> hotfixes;
+            const auto callback
+            {
+                [&key, &subKey, &hotfixes](const std::string & packageKey)
+                {
+                    const auto callbackKey
+                    {
+                        [&key, &subKey, &packageKey, &hotfixes](const std::string & package)
+                        {
+
+                            if (Utils::startsWith(package, "InstallProperties"))
```

Comment:
We have two places here to read the patches:

- `InstallProperties -> DisplayName` where we can find patches for the own OS
- `Patches -> <ID> -> DisplayName` where are located the patches for the Microsoft software

You can see an example in the issue description [(#6871)](https://github.com/wazuh/wazuh/issues/6871). The second-mentioned keys are not being read.

![image](https://user-images.githubusercontent.com/29475387/135122629-c5305db6-73b0-46d5-a5cf-46cbac299af6.png)